pkg/testdata/property-history-values.json

```diff
@@ -1,459 +1,464 @@
-                "TimeInSeconds": 1612205100
-            },
-            "Value": {
-                "BooleanValue": null,
-                "DoubleValue": 24.92403230837538,
-                "IntegerValue": null,
-                "StringValue": null
-            }
-        },
-        {
-            "Quality": "GOOD",
-            "Timestamp": {
-                "OffsetInNanos": 0,
-                "TimeInSeconds": 1612205400
-            },
-            "Value": {
-                "BooleanValue": null,
-                "DoubleValue": 24.488252036044447,
-                "IntegerValue": null,
-                "StringValue": null
-            }
-        },
-        {
-            "Quality": "GOOD",
-            "Timestamp": {
-                "OffsetInNanos": 0,
-                "TimeInSeconds": 1612205700
-            },
-            "Value": {
-                "BooleanValue": null,
-                "DoubleValue": 24.713524629788896,
-                "IntegerValue": null,
-                "StringValue": null
-            }
-        },
-        {
-            "Quality": "GOOD",
-            "Timestamp": {
-                "OffsetInNanos": 0,
-                "TimeInSeconds": 1612206000
-            },
-            "Value": {
-                "BooleanValue": null,
-                "DoubleValue": 24.062599503697264,
-                "IntegerValue": null,
-                "StringValue": null
-            }
-        },
-        {
-            "Quality": "GOOD",
-            "Timestamp": {
-                "OffsetInNanos": 0,
-                "TimeInSeconds": 1612206300
-            },
-            "Value": {
-                "BooleanValue": null,
-                "DoubleValue": 24.46870597335279,
-                "IntegerValue": null,
-                "StringValue": null
-            }
-        },
-        {
-            "Quality": "GOOD",
-            "Timestamp": {
-                "OffsetInNanos": 0,
-                "TimeInSeconds": 1612206600
-            },
-            "Value": {
-                "BooleanValue": null,
-                "DoubleValue": 24.851267281740665,
-                "IntegerValue": null,
-                "StringValue": null
-            }
-        },
-        {
-            "Quality": "GOOD",
-            "Timestamp": {
-                "OffsetInNanos": 0,
-                "TimeInSeconds": 1612206900
-            },
-            "Value": {
-                "BooleanValue": null,
-                "DoubleValue": 24.21436429829983,
-                "IntegerValue": null,
-                "StringValue": null
-            }
-        },
-        {
-            "Quality": "GOOD",
-            "Timestamp": {
-                "OffsetInNanos": 0,
-                "TimeInSeconds": 1612207200
-            },
-            "Value": {
-                "BooleanValue": null,
-                "DoubleValue": 23.81007059955162,
-                "IntegerValue": null,
-                "StringValue": null
-            }
+    "SuccessEntries": [
```

Comment:
Looks like the response format change significantly... is this expected?

Reply:
yep, this is the structure of the batch API response from AWS